tracing-subscriber/src/reload.rs

```diff
@@ -11,9 +11,29 @@
 //!
 //! # Examples
 //!
-//! Reloading a [`Filtered`](crate::filter::Filtered) layer to change the filter at runtime.
-//!
+//! Reloading a global filtering layer:
+//! ```rust
```

Comment:
very minor nitpick: elsewhere we prefer a newline between text and examples:
```suggestion
//!
//! ```rust
```

Reply:
Ah, oops, I got that from https://github.com/tokio-rs/tracing/blob/f70116611cddf03167cb67749d24ed7c5cfe7156/tracing-subscriber/src/layer/mod.rs#L79 😅